### PR TITLE
new pictogram on landing page, new about text, new data privacy, new …

### PR DESCRIPTION
**Pull request by andyz02**
…footer text

### CODE REVIEW

File: backend/templates/dataprotection.html [hunk 1]
```diff
@@ -6,106 +6,152 @@
...
+          Email : <a href="mailto:info@match4healthcare.de">info@match4healthcare.de</a></p>
+        <p>Bei einem Besuch unserer Webseite verarbeiten wir personenbezogene Daten, die Ihr Browser an unseren Server &uuml;bermittelt. </p>
+        <h4><strong>Essentielle Datenerhebungen und Datenerhebungen im berechtigten Interesse</strong></h4>
+        <p><strong>Allgemeine Informationen </strong></p>
+        <p><strong>Zweck</strong><br>
+          Wenn Sie unsere Website besuchen, &uuml;bertr&auml;gt Ihr Webbrowser Nutzungsdaten. Diese sind technisch erforderlich, um einen Verbindungsaufbau zwischen zugreifendem Rechner und Server zu erm&ouml;glichen und Ihnen damit unser Webangebot bereitzustellen.</p>
+        <p><strong>Datenkategorien</strong><br>
+          Es werden folgende Daten erhoben:</p>
+        <ul>
+          <li>IP-Adresse</li>
```
Comment (Baschdl) [inline backend/templates/dataprotection.html:51]: Where do we save the IP?

File: backend/templates/dataprotection.html [hunk 2]
```diff
@@ -6,106 +6,152 @@
...
-                    <hr>
-                    <br>
-                    Quelle: Teile dieser Datenschutzerklärung wurden mit dem Datenschutz Generator von firmenwebseiten.at erstellt.<br>
-                </p>
-                <h5>Haftungsausschluss AGB/Datenschutz</h5>
-                <p>Wir verbinden Helfende mit Hilfesuchenden. Der Einsatz erfolgt in Eigenverantwortung und in Verantwortung der entsprechenden Einrichtung. match4healthcare übernimmt weder die Haftung noch etwaige notwendige Versicherungen für die Einsätze, die im Anschluss an ein matching durch match4healthcare entstehen. Ebenfalls erfolgt die Angabe der Informationen im Portal auf Eigenverantwortung. Qualifikationen und alle angegebenen Informationen der NutzerInnen müssen vor Einsatzbeginn von Seiten der Hilfesuchenden und Helfenden nochmals verifiziert werden.</p>
-            </div>
-        </div>
-
+  <div style="height:40px"></div>
```
Comment (Baschdl) [inline backend/templates/dataprotection.html:22]: Can we just change the data protection rules without notifying the users?

File: backend/templates/home.html [hunk 3]
```diff
@@ -131,22 +131,19 @@ <h2 class="text-white font-weight-light">{% blocktrans %}Wir sind eine{% endbloc
...
-          <iframe class="embed-responsive-item" src="https://www.youtube-nocookie.com/embed/g8-EqUU55Bo" allowfullscreen></iframe>
-        </div>
-      </div>
-    </div>
+   
+	<div class="row" style="margin-bottom: 75px;">
+	  		<div class="col-md-12 col-lg-8" style="text-align: left; margin: 0 auto;">
+			 <h2 style="text-align:center; font-size: 1.5rem; font-weight:600; margin-bottom: 15px">{% blocktrans %}Wie funktioniert{% endblocktrans %} <img src="{% static 'img/match4healthcare-logo-text.svg' %}" width="180px" style="vertical-align: baseline">?</h2>
+				<hr style="width:60%; margin: 0 auto; margin-bottom:25px; border-width: 2px; ">
+			<p style="text-align:center;">match4healthcare vernetzt Helfende und Hilfesuchende aus dem Gesundheitswesen anhand von Kriterien wie Qualifikation, Verfügbarkeit und Wunschtätigkeit auf der einen und Bedarf auf der anderen Seite. Die Bereitstellung der Plattform und die Durchführung der Vernetzung von Helfenden und Hilfesuchenden werden durch den ehrenamtlichen Einsatz vieler hilfsbereiter Freiwilliger ermöglicht und erfolgt unentgeltlich.</p>
```
Comment (Baschdl) [inline backend/templates/home.html]: Translation missing

**Review by Baschdl — CHANGES_REQUESTED** [review]

File: backend/templates/dataprotection.html (hunk 1 above)
Comment (andreaskind) [reply backend/templates/dataprotection.html:51]: The whole document is a product of a conversation between our legal counsellor and @maltezacharias . I guess we're not saving but collect it in the first place, which should be declared as well?

File: backend/templates/dataprotection.html (hunk 2 above)
Comment (andreaskind) [reply backend/templates/dataprotection.html:22]: Yes, last time i asked our legal advice, we can, but I'll double check

**Review by Baschdl — DISMISSED** [review]
We also need a translation of the pretty pictogram

**Comment by andreaskind** [comment]
@Baschdl pictogramm of the frontpage has been translated and added to the static/img folder

**Comment by bjrne** [comment]
The image translation logic works. Please check the english version again @andreaskind, I cropped and downsized it to be appropriate for a website. It still differs in fonts and colors from the german version though, but that wouldn't be an issue for me. Otherwise this should be good to go.

File: backend/templates/base.html [hunk 4]
```diff
@@ -133,8 +133,9 @@
     </p>
     <p>{% blocktrans %}Ein Projekt im Rahmen des{% endblocktrans %} <a href="https://wirvsvirushackathon.org/"
                     target="_blank">#WirVsVirus Hackathons</a>. <a
```
Comment (bjrne) [inline backend/templates/base.html]: Missing translation, will fix.

**Comment by kevihiiin** [comment]
> The image translation logic works. Please check the english version again @andreaskind, I cropped and downsized it to be appropriate for a website. It still differs in fonts and colors from the german version though, but that wouldn't be an issue for me. Otherwise this should be good to go.

The language settings for the image work nicely, but the translation for the rest of the site broke completely. The cookie is set correctly, will look into that after I reviewed all the changes
- - - 
Edit:
The problem seems to come from our latest staging branch. Because this is unrelated I will create a new issue / branch for the fix.

Edit 2:
I forgot to make & compile the message file. Issue resolved

File: backend/templates/terms-of-use.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+<p>Wir übernehmen ausdrücklich keine Gewähr für Richtigkeit, Vollständigkeit, Verlässlichkeit, Geeignetheit und Aktualität sowie Brauchbarkeit der Inhalte. Diese dienen ausschließlich der allgemeinen Information. Hinweise, Empfehlungen und Auskünfte sind unverbindlich. Jegliche Verwendung der angebotenen Inhalte geschieht auf eigenes Risiko. Insbesondere nimmt match4healthcare keine Prüfung der vom Nutzer angegebenen Qualifikationen vor.</p>
+
+<h4>Wie schützt match4healthcare Daten?</h4>
+
+<p>Zur Gewährleistung der Datenminimierung gibt match4healthcare keine Namen und Kontaktdaten von Helfenden an die Hilfesuchenden weiter. Die unmittelbare Kontaktaufnahme erfolgt stets durch den Helfenden. Die Darstellung der Helfenden findet auf der Helferkarte lediglich in Gestalt der Anzahl der Helfenden in einem bestimmten Postleitzahlenbereich statt. In den Hilfesuchenden in ihrem Nutzerkonto zur Verfügung gestellten Listen werden darüber hinaus Qualifikationen, die Verfügbarkeit und mitgeteilt, ob der Helfende zu eine Unterkunft wünscht und eine Bezahlung benötigt.</p>
```
Comment (kevihiiin) [inline backend/templates/terms-of-use.html]:
```suggestion
<p>Zur Gewährleistung der Datenminimierung gibt match4healthcare keine Namen und Kontaktdaten von Helfenden an die Hilfesuchenden weiter. Die unmittelbare Kontaktaufnahme erfolgt stets durch den Helfenden. Die Darstellung der Helfenden findet auf der Helferkarte lediglich in Gestalt der Anzahl der Helfenden in einem bestimmten Postleitzahlenbereich statt. In den Hilfesuchenden zur Verfügung gestellten Listen über potentielle Helfende werden darüber hinaus Qualifikationen, Verfügbarkeit sowie ein eventueller Wunsch nach Unterkunft und Bezahlung der Hilfe bietenden mitgeteilt.</p>
```
The last sentence was a bit hard to understand, maybe this makes it clearer

File: backend/templates/impressum.html [hunk 6]
```diff
@@ -27,10 +27,10 @@ <h1>Impressum</h1>
         <div class="row">
             <div class="col-md-4 m-1 p-3">
                 <h5>Personen hinter dem Projekt</h5>
-                <strong>Amandeep Grewal, Andreas Zehetner, Ahmed Abdel-Rahman, Michael Neulinger und Alexandra Diendorfer</strong><br><br>
+                <strong>match4healthcare e.V. (in Gründung)</strong><br><br>
```
Comment (kevihiiin) [inline backend/templates/impressum.html]:
```suggestion
                <strong>match4healthcare e.V.</strong><br><br>
```
Are we still "in Gründung"?

**Review by kevihiiin — APPROVED** [review]

File: backend/templates/terms-of-use.html (hunk 5 above)
Comment (bjrne) [reply backend/templates/terms-of-use.html]: Hm, finds dann fast verständlicher wenns "In den *den* Hilfesuchenden" zur Verfügung [...]" ist :see_no_evil: Ist mir persönlich aber egal.

File: backend/templates/impressum.html (hunk 6 above)
Comment (bjrne) [reply backend/templates/impressum.html]: Not sure, further down it also just says "match4healthcare e.V.". @andreaskind?

File: backend/templates/dataprotection.html (hunk 1 above)
Comment (Baschdl) [reply backend/templates/dataprotection.html:51]: @maltezacharias has to respond to this regarding the nginx but we don't look at it in the python code

File: backend/templates/terms-of-use.html (hunk 5 above)
Comment (Baschdl) [reply backend/templates/terms-of-use.html]: Ist "Zur Gewährleistung der Datenminimierung gibt match4healthcare keine Namen und Kontaktdaten von Helfenden an die Hilfesuchenden weiter." wirklich korrekt, wenn ein Helfender direkt einen Hilfesuchenden anschreibt?

Comment (kevihiiin) [reply backend/templates/terms-of-use.html]: > Hm, finds dann fast verständlicher wenns "In den _den_ Hilfesuchenden" zur Verfügung [...]" ist 🙈 Ist mir persönlich aber egal.

Gerne auch das:) Ich bin nur bin nur über den Satz "zu eine Unterkunft wünscht" gestolpert und hab ihn nicht ganz verstanden. Ich glaube "zu eine[r] Unterkunft wünscht" passt auch.

Comment (bjrne) [reply backend/templates/terms-of-use.html]:
```suggestion
<p>Zur Gewährleistung der Datenminimierung gibt match4healthcare keine Namen und Kontaktdaten von Helfenden an die Hilfesuchenden weiter. Die unmittelbare Kontaktaufnahme erfolgt stets durch den Helfenden. Die Darstellung der Helfenden findet auf der Helferkarte lediglich in Gestalt der Anzahl der Helfenden in einem bestimmten Postleitzahlenbereich statt. In den den Hilfesuchenden zur Verfügung gestellten Listen über potentielle Helfende werden darüber hinaus Qualifikationen, Verfügbarkeit sowie ein eventueller Wunsch nach Unterkunft und Bezahlung der Hilfebietenden mitgeteilt.</p>
```

**Pull request merged**